aries-site/src/layouts/content/ExampleControls.js

```diff
@@ -1,77 +1,73 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
-import { Box, Button } from 'grommet';
+import { Box, Button, Grid, ResponsiveContext } from 'grommet';
 import { Document, Expand, Grommet, Figma } from 'grommet-icons';
 
-const ControlButton = ({ children, ...rest }) => (
-  <Button plain {...rest}>
-    {({ hover }) => (
-      <Box
-        direction="row"
-        align="center"
-        gap="small"
-        pad={{ vertical: 'xsmall', horizontal: 'small' }}
-        round="small"
-        background={hover ? 'active-background' : undefined}
-      >
-        {children}
-      </Box>
-    )}
-  </Button>
-);
-
-ControlButton.propTypes = {
-  children: PropTypes.oneOfType([PropTypes.element, PropTypes.array])
-    .isRequired,
-};
+export const ExampleControls = ({ designer, docs, figma, setShowLayer }) => {
+  const size = useContext(ResponsiveContext);
+  const buttonSize = size !== 'small' ? undefined : 'small';
 
-export const ExampleControls = ({ designer, docs, figma, setShowLayer }) => (
-  <Box
-    background="background-front"
-    border={{
-      side: 'top',
-      color: 'background-back',
-      size: 'xsmall',
-    }}
-    direction="row"
-    align="start"
-    justify="between"
-    pad={{ horizontal: 'medium', vertical: 'small' }}
-    round={{ corner: 'bottom', size: 'small' }}
-  >
-    <Box direction="row-responsive" gap="small">
-      {designer && (
-        <Button
-          href={designer}
-          icon={<Grommet color="plain" />}
-          label="Open in Grommet"
-          target="_blank"
-        />
-      )}
-      {figma && (
-        <Button
-          href={figma}
-          icon={<Figma color="plain" />}
-          label="Open in Figma"
-          target="_blank"
-        />
-      )}
-      {docs && (
+  return (
+    <Box
+      direction="row"
+      align="start"
+      background="background-front"
+      border={{
+        side: 'top',
+        color: 'background-back',
+        size: 'xsmall',
+      }}
+      justify="between"
+      pad={{ horizontal: 'medium', vertical: 'small' }}
+      round={{ corner: 'bottom', size: 'small' }}
+    >
+      <Grid
+        columns={{ count: 'fill', size: ['small', 'auto'] }}
```

Comment:
The responsiveness is better on the most recent, but the spacing between the buttons on the left isn't uniform:
<img width="1398" alt="Screen Shot 2020-06-09 at 9 21 06 AM" src="https://user-images.githubusercontent.com/12522275/84173898-bf223c00-aa32-11ea-8eb3-23b99c1e098e.png">

"Open docs" is further away from the others